tests/cortex_cli_auth_login_test.py

```diff
@@ -329,3 +329,40 @@ def test_auth_login_starts_token_manager(config_dict, credentials):
             assert 'Starting token manager daemon' in result.output
 
     unstub()
+
+
+def test_auth_login_no_daemonization_on_windows(config_dict, credentials):
+    """
+    Tests that damonization is disabled on windows, even if not testing on windows.
```

Comment:
```suggestion
    Tests that daemonization is disabled on Windows, even if not testing on Windows.
```

Reply:
Applied the suggestion. Yes, no new version, just fixing tests.